src/OpenSage.Game/Logic/Player.cs

```diff
@@ -1,4 +1,4 @@
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
```

Comment:
yeah, i can't find out why git sees this as a change. maybe this has to do something with CRLF\LF line endings. anyway i'll just restore this file

Reply:
Guessing it's a BOM issue. I see these often in commits, mine included 🤷 

If you're already working in the file, it is what it is. Otherwise, yeah I'd just restore any otherwise unchanged files.